dask_cuda/proxify_host_file.py

```diff
@@ -105,12 +127,19 @@ class ProxiesOnHost(Proxies):
     This uses dask.sizeof to update memory usage.
     """
 
-    def mem_usage_add(self, proxy: ProxyObject):
+    def mem_usage_add(self, proxy: ProxyObject) -> None:
         self._mem_usage += sizeof(proxy)
 
-    def mem_usage_remove(self, proxy: ProxyObject):
+    def mem_usage_remove(self, proxy: ProxyObject) -> None:
         self._mem_usage -= sizeof(proxy)
 
+    def buffer_info(self) -> List[Tuple[float, int, List[ProxyObject]]]:
+        ret = []
+        for p in self.get_proxies():
+            size = sizeof(p)
+            ret.append((p._pxy_get().last_access, size, [p]))
```

Comment:
Why is a `List[ProxyObject]` needed here instead of just `ProxyObject`? It seems like it's always a single-element list.